pkg/container/client.go

```diff
@@ -119,7 +119,12 @@ func (client dockerClient) GetContainer(containerID string) (Container, error) {
 		return Container{}, err
 	}
 
-	container := Container{containerInfo: &containerInfo}
+	imageInfo, _, err := client.api.ImageInspectWithRaw(bg, containerInfo.Image)
```

Comment:
@simskij the crash found in #215 is actually caused by `client.api.ImageInspectWithRaw`, _not_ by `ContainerJSONBase.Image` -> `container.imageInfo.ID` needs `ImageInspectWithRaw`.

Alternatively, if we handle the error from `ImageInspectWithRaw` properly (a log message along the lines of "The source image for container x is not present locally, cant update, skipping...") we might have the most reasonable solution?

Reply:
I changed to a warning and then handle imageInfo being nil in `Update`, skipping the container in such an event.